backend/src/test/java/com/ambacam/ItBase.java

```diff
@@ -1,45 +1,50 @@
 package com.ambacam;
 
-import java.util.Random;
-
+import com.ambacam.model.Action;
+import com.ambacam.model.Role;
+import com.ambacam.model.StatusRequete;
+import io.restassured.RestAssured;
 import org.junit.runner.RunWith;
 import org.springframework.boot.context.embedded.LocalServerPort;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
 import org.springframework.test.context.junit4.SpringRunner;
 
-import com.ambacam.model.Role;
-import com.ambacam.model.StatusRequete;
-
-import io.restassured.RestAssured;
+import java.util.Random;
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
 public class ItBase {
-	protected Random random = new Random();
+    protected Random random = new Random();
+
+    @LocalServerPort
+    protected int port;
+
+    public void setup() throws Exception {
+        RestAssured.port = port;
 
-	@LocalServerPort
-	protected int port;
+    }
 
-	public void setup() throws Exception {
-		RestAssured.port = port;
+    public void cleanup() throws Exception {
 
-	}
+    }
 
-	public void cleanup() throws Exception {
+    protected Role buildRole() {
 
-	}
+        Role role = new Role().nom("nom-" + random.nextLong()).description("description-" + random.nextLong());
+        return role;
+    }
 
-	protected Role buildRole() {
+    protected StatusRequete buildStatusRequete() {
 
-		Role role = new Role().nom("nom-" + random.nextLong()).description("description-" + random.nextLong());
-		return role;
-	}
+        StatusRequete statusRequete = new StatusRequete().nom("nom-" + random.nextLong())
+                .description("description-" + random.nextLong());
+        return statusRequete;
+    }
 
-	protected StatusRequete buildStatusRequete() {
+    protected Action buildAction() {
 
-		StatusRequete statusRequete = new StatusRequete().nom("nom-" + random.nextLong())
-				.description("description-" + random.nextLong());
-		return statusRequete;
-	}
+        Action action = new Action().nom("nom-" + random.nextLong());

```

Comment:
Please set description also

Reply:
Ok I will add it. I got Action attributes from the Technical specifications docs. It seems that it is not up to date.